Dockerfile

```diff
@@ -32,7 +31,7 @@ RUN apt-get install --no-install-recommends -y \
 # Install composer
 RUN curl https://getcomposer.org/installer | php -- && mv composer.phar /usr/local/bin/composer && chmod +x /usr/local/bin/composer
 
-RUN composer config -g github-oauth.github.com $git_oauth_token
+RUN composer
```

Comment:
I think you can remove this step completely since it was just setting the config rather than doing an install.

Reply:
👍